### DISCUSSION TITLE
How to create text with the kbd style / format

### DISCUSSION THREAD

**Discussion by antamy** [TriliumNext/Trilium#4579] [Q&A]
I would like to create text similar to what is in the Trillium help page, where keyboard shortcuts are shown.  This text is surrounded by kbd /kbd in the HTML.  How is this done in the Trillium editor?

![image](https://github.com/zadam/trilium/assets/6841038/954bec8b-67a7-4154-bcc3-25a11e603777)

**Comment by meichthys** [comment]
Use ` before and after the text.

**Reply by antamy** [reply]
That formats it as code - it just changes the background and results in code tags being used, not kbd tags.  I am using the out-of-the-box dark theme.  What I would like to achieve is the following type of styling with the rounded edges and the border:

code: <code>CTRL-Z</code>, kbd: <kbd>CTRL-Z</kbd>

I am assuming this is built-in, since supplied pages display these, I just can't figure out how to format text like that.

**Reply by meichthys** [reply]
Interesting. I didn't even know about kbd tags lol. I don't think this is something that is supported in the editor since Trilium uses ckeditor5 which doesn't support this yet: https://github.com/ckeditor/ckeditor5/issues/12494

**Reply by zadam** [reply]
Not all tags are supported in CKEditor. `<kbd>` is one of those unsupported and given it's so niche, I don't expect first class support.

**Reply by antamy** [reply]
ok, but how did you create it in the Trilium help page in my original post?
